openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -165,25 +165,21 @@ class Meta(object):  # pylint: disable=missing-docstring
         read_only_fields = ()
         explicit_read_only_fields = ("profile_image", "requires_parental_consent")
 
-    def validate_name(self, attrs, source):
+    def validate_name(self, new_name):
         """ Enforce minimum length for name. """
-        if source in attrs:
-            new_name = attrs[source].strip()
-            if len(new_name) < NAME_MIN_LENGTH:
-                raise serializers.ValidationError(
-                    "The name field must be at least {} characters long.".format(NAME_MIN_LENGTH)
-                )
-            attrs[source] = new_name
+        if len(new_name) < NAME_MIN_LENGTH:
+            raise serializers.ValidationError(
+                "The name field must be at least {} characters long.".format(NAME_MIN_LENGTH)
```

Comment:
This message will get seen by a user if they try to put in a name that is too short, right? If so, then it needs to be internationalized (wrapped in `ugettext` calls).